### ISSUE TITLE
Storage bar not filling in the redesigned account page

### ISSUE THREAD

**Issue by mbommerez** [web3-storage/web3.storage#1157]

**Comment by mbommerez** [comment]
@orvn Assigned to you as I think you are running point on this. Feel free to unassign yourself if it is not. Or mark as duplicate if it is already tracked somewhere else. Thanks!

**Comment by orvn** [comment]
Thanks @mbommerez! You have the correct person 🙂

I think we might be rounding the value down: will make sure the first bit is stepped to like 2% or something low like that, so that it's always visible if someone has stored any quantity of files.

**Comment by dchoi27** [comment]
@mbommerez Separate from this issue, do we have an issue tracking getting the updated storage limits from the database shown in the website (so folks can now see their updated storage limits, rather than it showing 1TiB for everyone)? Didn't see anything tracking that, but you can loop @orvn in to get that done.

**Comment by orvn** [comment]
@dchoi27 is that for the case where users are granted storage `> 1 TiB`?

If so, I guess my question is: do we show any upper limit once they've crossed `1 TiB` at all? We did away with the `10 TiB` cap, which was the previous ceiling.

**Comment by dchoi27** [comment]
Yes, if the user has an increased storage capacity. I think it's just about making the bar scale to whatever the user's current storage limit is (1TiB by default, but higher if they were granted more). 
<img width="631" alt="image" src="https://user-images.githubusercontent.com/11778450/159981229-1269afc1-2aec-402f-be0c-65747d2ab028.png">

I'm not sure where we landed on the previous discussion, but I had mentioned that we don't need the two links to the storage increase request form (currently the "🔒 1TiB+" and the "Submit a request" links both link to the request form). I think what we can do is get rid of the "🔒 1TiB+" link completely so it's generalizable (no matter what their storage limit is, we just have to modify the number at the right end of the bar and have the fill scale to that number).

**Comment by orvn** [comment]
> I'm not sure where we landed on the previous discussion, but I had mentioned that we don't need the two links to the storage increase request form (currently the "🔒 1TiB+" and the "Submit a request" links both link to the request form)

That's right, we did remove the link at the bottom, but restored the link under the bar. Will remove the one attached to the lock symbol.

> I'm not sure where we landed on the previous discussion, but I had mentioned that we don't need the two links to the storage increase request form (currently the "🔒 1TiB+" and the "Submit a request" links both link to the request form)

Do you think it should just fade away at the right end? So as to indicate that users in the higher tier don't have a defined upper limit?

**Comment by dchoi27** [comment]
hey @orvn - where are we on this?

**Comment by orvn** [comment]
@dchoi27, just some clarification on this issue, the storage bars are indeed filling up. It's just < 1px in width when it's under `1024 GiB / 512 px` = `2 GiB`. But two PRs help resolve this: #1136 (which increases the desktop pixel width of the bar) and #1288, which splits the storage usage bar and sets a minimum visual size.
